src/activemasternode.cpp

```diff
@@ -205,3 +106,130 @@ bool CActiveMasternode::EnableRemoteMasterNode(CTxIn& vinNew, CService& serviceN
 
     return true;
 }
+
+void CActiveMasternode::ManageStateInitial()
+{
+    if(!pwalletMain) {
+        strNotCapableReason = "Wallet not available.";
+        LogPrintf("CActiveMasternode::ManageStateInitial -- not capable: %s\n", strNotCapableReason);
+        return;
+    }
+
+    if(pwalletMain->IsLocked()) {
+        strNotCapableReason = "Wallet is locked.";
+        LogPrintf("CActiveMasternode::ManageStateInitial -- not capable: %s\n", strNotCapableReason);
+        return;
+    }
+
+    if(pwalletMain->GetBalance() == 0) {
```

Comment:
I remembered seeing this problem and I thought I had done something about it, but I guess not.

Curiously I have a remote MN running this code now that appears to be working, right down to the expected log messages.  I'm a bit puzzled by why that is, but I'll fix this somehow.